dotCMS/src/main/java/com/dotcms/contenttype/transform/field/JsonFieldTransformer.java

```diff
@@ -44,9 +47,9 @@ public JsonFieldTransformer(String json) {
       }
     } catch (Exception e) {
       try {
-      	JSONObject jo = new JSONObject(json);
-        if (jo.has("fields")) {
-          l = fromJsonArray(jo.getJSONArray("fields"));
+        final JSONObject fieldJsonObjec = new JSONObject(json);
+        if (fieldJsonObjec.has("fields")) {
+          l = fromJsonArray(fieldJsonObjec.getJSONArray("fields"));
```

Comment:
1) I think it is a type fieldJsonObjec should be fieldJsonObject
2) rename l to something meaninful